src/sentry/notifications/utils/participants.py

```diff
@@ -269,10 +269,19 @@ def get_release_committers(project: Project, event: Event) -> Sequence[User]:
 
     # commit_author_id : Author
     author_users: Mapping[str, Author] = get_users_for_commits(commits)
-    return list(
+    release_committers = list(
         User.objects.filter(id__in={au["id"] for au in author_users.values() if au.get("id")})
     )
 
+    return list(
```

Comment:
Let's add a comment here just noting that this is inefficient since it hits flagr once per user. Since this is isolated to sentry users it's fine, but just want that to be apparent if anyone else sees this code and wants to copy it.